Modules/@babylonjs/react-native/ios/EngineViewManager.mm

```diff
@@ -55,6 +66,10 @@ - (void)drawRect:(CGRect)rect {
     if ([BabylonNativeInterop isXRActive]) {
         if (!xrView) {
             xrView = [[MTKView alloc] initWithFrame:self.bounds device:self.device];
+            if(self.isTransparent == YES){
+                [xrView setOpaque:NO];
+                [xrView.window setOpaque:NO];
+            }
```

Comment:
This should not be needed, right? Any transparency settings should never affect the xr view, right?

Reply:
Removed and tested.